requirements.txt

```diff
@@ -1,12 +1 @@
-git+git://github.com/googleapis/python-storage@8cf6c62a96ba3fff7e5028d931231e28e5029f1c
-grpcio==1.38.1
-grpcio-tools==1.38.1
-googleapis-common-protos==1.53.0
-protobuf==3.17.3
-flask==2.0.1
-httpbin==0.7.0
-requests-toolbelt==0.9.1
-scalpl==0.4.2
-crc32c==2.2
-gunicorn==20.1.0
-schema==0.7.4
+.
```

Comment:
Thanks for adding the support files! Will we be removing this `requirements.txt` file entirely once finalized? (Meaning that all users should install the testbench with `pip`?)  If so, we'll also need to update the readme instructions.

Reply:
I honestly do not know.  We could change the instructions in this PR, I would be Okay with that.  I am Okay with pushing to a future PR.

Right now, `pip install -e .` works: it installs the dependencies needed for development.  But arguably `pip install -r requirements.txt` installs too much.

Reply:
That's good to know, thanks! I'll have to look into pip more. I agree we can move this forward and push instruction changes to a future PR.